docs/resources/sfs_turbo_du_task.md

```diff
@@ -0,0 +1,100 @@
+---
+subcategory: "Scalable File Service (SFS)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_sfs_turbo_du_task"
+description: ""
+---
+
+# huaweicloud_sfs_turbo_du_task
+
+Manages a DU task resource under the SFS Turbo within HuaweiCloud.
+
+-> SFS Turbo File systems created after August 1, 2023 support this resource.
+    If there are `10` executing tasks, no new tasks areallowed to be created.
+    It is recommended that no more than `4` query requests be submitted at the same time.
+    Querying a very large directory tasks a long time, please do not submit the request again.
```

Comment:
Please do not use loose English and do not explain too many words that have nothing to do with resources.

Reply:
Remove extra spaces.